posix.mak

```diff
@@ -13,6 +13,8 @@
 #
 # make BUILD=debug unittest => builds all unittests (for debug) and runs them
 #
+# make DEBUGGER=ddd BUILD=debug std/XXXXX.debug => builds the module XXXXX and executes it in the debugger ddd
```

Comment:
I have been working on that a little, and this would require quite a lot changes to the whole makefile